### ISSUE TITLE
Open Files Window with Xubuntu 17.10 German Locals

### ISSUE THREAD

**Issue by Meresmata** [LightTable/LightTable#2364]
### Steps to Reproduce

 1. _I downloaded the linux 64-bit version.
 2. _Started light table.
 3._Wanted open a file

 4._Got following Window (using Xubuntu 17.10 with German locals)
![bildschirmfoto_2018-03-11_23-05-51](https://user-images.githubusercontent.com/6282825/37259056-9d4a12f2-2581-11e8-9ce4-0949034c3d53.png)

### Environment

| Info                             | Details |
| -------------------------------- | ------- |
| Light Table version              |     0.81    |
| Operating system                 |      Xubuntu 17.10 64-bit   |

**Comment by sbauer322** [comment]
This like issue #2343, which is related to #2359 and #2360.

I am going to close this issue in favor of #2343, but should the problem persist after #2343 is resolved, please feel free to open another issue.
